front/app/containers/IdeasShow/components/IdeaSharingModal/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import usePhases from 'api/phases/usePhases';
+import { getInputTerm } from 'api/phases/utils';
+
+import SharingModalContent from 'components/PostShowComponents/SharingModalContent';
+import Modal from 'components/UI/Modal';
+
+import { useIntl } from 'utils/cl-intl';
+import { getInputTermMessage } from 'utils/i18n';
+
+import messages from './messages';
+
+interface Props {
+  projectId: string;
+  newIdeaId: string;
+  closeIdeaSocialSharingModal: () => void;
+}
+
+const IdeaSharingModal = ({
+  projectId,
+  newIdeaId,
+  closeIdeaSocialSharingModal,
+}: Props) => {
+  const { data: phases } = usePhases(projectId);
+  const { formatMessage } = useIntl();
+
+  return (
+    <Modal
+      opened={!!newIdeaId}
```

Comment:
Then since `newIdeaId` will always be there, we can just pass opened
```suggestion
      opened
```